server/routes/schedules.js

```diff
@@ -0,0 +1,21 @@
+const express = require("express");
+const router = express.Router();
+const conn = require("../mariadb");
+const { StatusCodes } = require("http-status-codes");
+const {
+  getAllSchedules,
+  getScheduleById,
+  addSchedule,
+  updateSchedule,
+  deleteSchedule,
+} = require("../controller/schedulesController");
```

Comment:
아래와 같이 require문을 먼저 모아놓는데, 그 중에서도 외부 라이브러리에서 불러오는 코드와 내가 작성한 코드는 개행으로 구분을 해주고 그 아래에 코드를 작성하면 좋을 것 같아요~
```suggestion
const express = require("express");
const { StatusCodes } = require("http-status-codes");

const conn = require("../mariadb");
const {
  getAllSchedules,
  getScheduleById,
  addSchedule,
  updateSchedule,
  deleteSchedule,
} = require("../controller/schedulesController");

const router = express.Router();
```